tests/acceptance/app-uses-helpers-test.js

```diff
@@ -0,0 +1,64 @@
+import Ember from 'ember';
+import { module, test } from 'qunit';
+import startApp from '../../tests/helpers/start-app';
+import {
+  clickOn,
+  clickRole,
+  fillInField,
+  findRole,
+} from 'ember-tb-test-helpers';
+
+module('Acceptance | App uses helpers', {
+  beforeEach() {
+    this.application = startApp();
+  },
+
+  afterEach() {
+    Ember.run(this.application, 'destroy');
+  }
+});
+
+test('finders', assert => {
+  visit('/');
+
+  clickOn('Checkbox!');
+  clickOn('Submit!');
+  clickRole('outer', 'middle', 'inner', 'checkbox');
+  fillInField('the-input', 'fillInField');
+
+  andThen(() => {
```

Comment:
'arrow function syntax (=>)' is only available in ES6 (use esnext option).<br>'andThen' is not defined.